lib/icinga/legacytimeperiod.cpp

```diff
@@ -122,7 +122,7 @@ boost::gregorian::date LegacyTimePeriod::GetEndOfMonthDay(int year, int month)
 
 void LegacyTimePeriod::ParseTimeSpec(const String& timespec, tm *begin, tm *end, tm *reference)
 {
-	/* Let mktime() figure out whether we're in DST or not. */
+	/* Let Utility::MkTime() figure out whether we're in DST or not. */
 	reference->tm_isdst = -1;
```

Comment:
The comment now makes little sense as that's what `Utility::MkTime()` always does. There still are more places where `tm_isdst` is explicitly set.

Reply:
Due to the fragility of this area I'd like to follow the principle: "Never touch a ..."